ee/clickhouse/queries/funnels/funnel_time_to_convert.py

```diff
@@ -1,12 +1,22 @@
-from ee.clickhouse.queries.funnels.funnel import ClickhouseFunnelNew
+from typing import Type
 
+from ee.clickhouse.queries.funnels.base import ClickhouseFunnelBase
+from posthog.models.filters.filter import Filter
+from posthog.models.team import Team
+
+
+class ClickhouseFunnelTimeToConvert(ClickhouseFunnelBase):
+    def __init__(self, filter: Filter, team: Team, funnel_order_class: Type[ClickhouseFunnelBase]) -> None:
+        super().__init__(filter, team)
+        self.funnel_order = funnel_order_class(filter, team)
```

Comment:
this is the composition. Every visualization type expects an ordering type via which it generates its inner query.